tests/conftest.py

```diff
@@ -35,52 +35,52 @@ def expected_csv_metadata():
 def convo_utts():
```

Comment:
Does this actually contain cases for testing the cleaning regexes?

Reply:
Yes! In the test_utterance file, `test_postinit_utterance()`